### ISSUE TITLE
PnP modern search web part: Placeholder text (Grey) is not WCAG compliant and not configurable.

### ISSUE THREAD

**Issue by willjohns** [microsoft-search/pnp-modern-search#4446]
**Version Used:** 
4.16.0.0

**Describe the bug:**
Using the PnP Search add-in, there is no ability to change the placeholder text colour (grey).
Grey (#878787) search text and the white (#FFFFFF) background, and does not pass WCAG standards.

**To Reproduce**
Import the PnP Search Web Part into SharePoint Site Page and configure as appropriate. (The search box text colour is not configurable)

**Additional Context:**
Please help me fix / setup.

**Comment by kasperbolarsen** [comment]
Please be aware that we can only support the current version,  which is 4.16.
Please update to this version and let us know if the issue has been resolved or not

**Comment by willjohns** [comment]
I have upgraded the web part to 4.9.3.0 and the issue persists..

**Comment by willjohns** [comment]
Thanks @kasperbolarsen can you confirm next steps?

**Comment by kasperbolarsen** [comment]
Next step would be for you to verify if the issue is there in v4.16, preferable in multi tenants

**Comment by willjohns** [comment]
Thanks @kasperbolarsen issue is indeed present in 4.16

**Comment by kasperbolarsen** [comment]
Can you lookup the settings for the search box in the Dev tools? The color is black in the 3 tenants I have 

<img width="340" height="770" alt="Image" src="https://github.com/user-attachments/assets/709fe776-3ce2-480c-b0d0-9eb3e7892eca" />

**Comment by willjohns** [comment]
![Image](https://github.com/user-attachments/assets/f809171f-adab-4990-8a14-b51ef6037d8c)

**Comment by willjohns** [comment]
Thanks @kasperbolarsen not sure if it's helpful but have provided the DevTools from settings. We embed the content in sharepoint so i presume that it's picked up a background colour somewhere? can you confirm as it's not inside our Theme..

**Comment by willjohns** [comment]
Thanks Kasper, i have obtained this and attached: Thanks for your help.
![Image](https://github.com/user-attachments/assets/4050b995-80fa-438a-97d1-9e95239974d7)

**Comment by kasperbolarsen** [comment]
So the color is black :

<img width="257" height="72" alt="Image" src="https://github.com/user-attachments/assets/2cc89688-b888-42db-a87c-411af1202253" />

Or is the issue something completely different?

**Comment by willjohns** [comment]
Thanks @kasperbolarsen ,

Apologies i'm not asking about the main text colour which is configured as black.
Can you confirm where the "placeholder text" colour is configured within your web part? 
Thanks in advance.

<img width="1784" height="671" alt="Image" src="https://github.com/user-attachments/assets/a1e925c2-a358-487f-81e4-7548aec87817" />

**Comment by kasperbolarsen** [comment]
Hi, while looking at the code , https://github.com/microsoft-search/pnp-modern-search/blob/main/search-parts/src/webparts/searchBox/components/SearchBoxContainer.module.scss  it seems like we are not defining the placeholder text color, but just using the default, as per:

https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder

**Comment by kasperbolarsen** [comment]
We very much welcome Pull Requests and will be looking forward to receiving one or more from you 😀

**Comment by kasperbolarsen** [comment]
Closed due to lack of activity
